examples/cra/jest.config.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import type { Config } from "@jest/types";
+import { pathsToModuleNameMapper } from "ts-jest";
+// Load the config which holds the path aliases.
+import { compilerOptions } from "./tsconfig.paths.json";
+
+const config: Config.InitialOptions = {
+  preset: "ts-jest",
+  testEnvironment: "jsdom",
+  moduleNameMapper: pathsToModuleNameMapper(compilerOptions.paths, {
+    // This has to match the baseUrl defined in tsconfig.json.
+    prefix: "<rootDir>/../../",
+  }),
+};
+
+export default config;
```

Comment:
It looks like `craco` doesn't like an external jest config, but you can put the contents of it in [`craco.config.js#jest#configure`](https://github.com/gsoft-inc/craco/blob/master/packages/craco/README.md#configuration-file).

Reply:
Would have never known.  Thanks so much.  Helped remove the transformignorepatterns junk in the package.json